widgets/AnaplanFunctionRunner-action.json

```diff
@@ -0,0 +1,89 @@
+{
+  "metadata": {
+    "spec-version": "1.4"
+  },
+  "display-name": "Anaplan Function",
+  "configuration-groups": [
+    {
+      "label": "Function Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Workspace Id",
+          "name": "workspaceId",
+          "widget-attributes": {
+            "width": "large"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Model Id",
+          "name": "modelId",
+          "widget-attributes": {
+            "width": "large"
+          }
+        },
+        {
+          "widget-type": "select",
```

Comment:
when there are only a couple options, we usually use the 'radio-group' widget. See https://github.com/cdapio/hydrator-plugins/blob/develop/core-plugins/widgets/File-batchsource.json#L162 for an example.

Reply:
Thanks! Updated